Gems/Atom/RPI/Code/Include/Atom/RPI.Public/FeatureProcessor.h

```diff
@@ -87,6 +89,9 @@ namespace AZ
             //! Perform any necessary deactivation.
             virtual void Deactivate() {}
 
+            //! Apply changes to render pipeline from feature processors
```

Comment:
nit: it's a good description, but I would connect it to passes for clarity - something like:
   'Apply changes and add additional render passes the the render pipeline from the feature processors'